pubspec.yaml

```diff
@@ -81,6 +79,7 @@ flutter:
   # To add assets to your application, add an assets section, like this:
   assets:
     - assets/
+    - assets/default_values/remote_config_default_values.json
```

Comment:
are you sure you still need that ?

Reply:
Yes. I tried to delete it and the assets couldn't be retrieved ..